test/integration/golden_response/get_statvar_summary/total_crimes.json

```diff
@@ -82,6 +82,126 @@
             }
           ]
         }
+      },
+      "provenanceSummary": {
```

Comment:
@n-h-diaz Please confirm the cache change

Reply:
this looks okay, thanks!